ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandDeduplicationIT.scala

```diff
@@ -294,13 +294,13 @@ final class CommandDeduplicationIT(
           acceptedLedgerOffset,
         ).map(_ => None)
       else
-        submitRequestAndAssertDeduplication(
+        submitRequestAndAssertAsyncDeduplication(
```

Comment:
Since the distinction no longer exists, why not rename `submitRequestAndAssertAsyncDeduplication` -> `submitRequestAndAssertDeduplication`. It would reduce the amount of visual noise

Reply:
Indeed. Amended